webapp/src/ts/services/settings.service.ts

```diff
@@ -29,7 +29,7 @@ export class SettingsService {
     });
   }
 
-  get() {
+  get():Promise<{ [key: string]: any }> {
```

Comment:
```suggestion
  get():Promise<Record<string, any>> {
```
Another alternative, both works